### PR TITLE
Make more natural English

### PR DESCRIPTION
**Pull request by cosmo0920**
- Use we glad to ~ for the annouce opening sentence
- can visualize -> makes to be able to visualize
- Use plural form for 'Feature'
- is included -> contains
  - Because in English, An inianimate object can become subject of a sentence.
- Add missing subjects
  - e.g. 'this plugin'
- Excepting ~ excluding -> Except for ~ not covered
- Add missing relative pronouns
  - e.g. table 'that' contains logs of every fixed time
- More concrete section name
  - Future -> Plan for the Future
- for details -> in more details
- Add missing _who_ do
  - e.g. 'We welcome feedback' is missing who do feedback into us.
  - 'We welcome _your_ feedback' is a perfect sentence. Oh, close! =)
- Add missing who doing
  - e.g. Introduced the first ... -> We announced that the first ...
- More concrete sentence
  - Let's visualize Groonga -> Let's visualize Groonga's data with the new Grafina data source

Follows up this commit: https://github.com/groonga/groonga.org/commit/dc4cdca64e0ce2014e5daa1dee9169f7bdc46bc7

### CODE REVIEW

**Comment by cosmo0920** [comment]
I tried to make more natural English in Grafina data source post.
Could you review this PR, @myokoym ?

File: en/_posts/2016-02-29-grafana-datasource-plugin-groonga.md [hunk 1]
```diff
@@ -6,37 +6,37 @@ description: grafana-datasource-plugin-groonga 0.1.0 has been released!
 
 ## grafana-datasource-plugin-groonga 0.1.0 has been released
 
-[grafana-datasource-plugin-groonga](https://github.com/groonga/grafana-datasource-plugin-groonga) 0.1.0 released. It's the first version of grafana-datasource-plugin-groonga.
+We glad to announce [grafana-datasource-plugin-groonga](https://github.com/groonga/grafana-datasource-plugin-groonga) 0.1.0 release. It's the first release of grafana-datasource-plugin-groonga.
```
Comment (cosmo0920) [inline en/_posts/2016-02-29-grafana-datasource-plugin-groonga.md:9]: We glad to 〜などを使って、嬉しい感情を表現しておいた方がなんとなく読みやすいです。

File: en/_posts/2016-02-29-grafana-datasource-plugin-groonga.md [hunk 2]
```diff
@@ -6,37 +6,37 @@ description: grafana-datasource-plugin-groonga 0.1.0 has been released!
...
 
-grafana-datasource-plugin-groonga is a plugin for [Grafana](http://grafana.org/) that adds a datasource for Groonga. It can visualize data of Groonga.
+grafana-datasource-plugin-groonga is a plugin for [Grafana](http://grafana.org/) that adds a datasource for Groonga. It makes to be able to visualize data of Groonga easily.
 
-### Feature
+### Features
```
Comment (cosmo0920) [inline en/_posts/2016-02-29-grafana-datasource-plugin-groonga.md:13]: Featureは複数あるので複数形にしておきたいです

File: en/_posts/2016-02-29-grafana-datasource-plugin-groonga.md [hunk 3]
```diff
@@ -6,37 +6,37 @@ description: grafana-datasource-plugin-groonga 0.1.0 has been released!
...
 
-* Visualize numeric column data of a table that is included Time type column.
+* Visualize numeric column data of a table that contains Time type column.
```
Comment (cosmo0920) [inline en/_posts/2016-02-29-grafana-datasource-plugin-groonga.md:15]: is includedというよりも「テーブルが」含むのでcontainsにしたいですね

File: en/_posts/2016-02-29-grafana-datasource-plugin-groonga.md [hunk 4]
```diff
@@ -6,37 +6,37 @@ description: grafana-datasource-plugin-groonga 0.1.0 has been released!
...
+Now, this plugin has only the above features. For example, you can visualize specific column data transition in a table that contains logs of every fixed time.
 
-### Future
+### Plan for the Future
 
 * Visualize multiple columns in a graph.
 * Graph results of drilldown.
 
-The above will be supported in future release.
+The above features will be supported in a future release.
```
Comment (cosmo0920) [inline en/_posts/2016-02-29-grafana-datasource-plugin-groonga.md:27]: ある未来のリリースというニュアンスを持たせたいのでfutureの前のaは必要です

File: en/_posts/2016-02-29-grafana-datasource-plugin-groonga.md [hunk 5]
```diff
@@ -6,37 +6,37 @@ description: grafana-datasource-plugin-groonga 0.1.0 has been released!
...
 
 ### Installation
 
-Copy a Git repository or archive of grafana-datasource-plugin-groonga to plugins directory in Grafana. Then, restart Grafana server. See [README](https://github.com/groonga/grafana-datasource-plugin-groonga#installation) for details.
+Copy a Git repository or archive of grafana-datasource-plugin-groonga into plugins directory in Grafana. Then, restart Grafana server. See [README](https://github.com/groonga/grafana-datasource-plugin-groonga#installation) in more details.
```
Comment (cosmo0920) [inline en/_posts/2016-02-29-grafana-datasource-plugin-groonga.md:31]: 〜の中に（すっぽりと）コピーする必要があるというように読み取れたのでtoではなくintoを使いたいです

File: en/_posts/2016-02-29-grafana-datasource-plugin-groonga.md [hunk 6]
```diff
@@ -44,10 +44,10 @@ See [README](https://github.com/groonga/grafana-datasource-plugin-groonga#usage)
 
 ### Feedback
 
-We welcome feedback via [Groonga-talk mailing list](https://lists.sourceforge.net/lists/listinfo/groonga-talk) or [GitHub Issues](https://github.com/groonga/grafana-datasource-plugin-groonga/issues).
+We welcome your feedback via [Groonga-talk mailing list](https://lists.sourceforge.net/lists/listinfo/groonga-talk) or [GitHub Issues](https://github.com/groonga/grafana-datasource-plugin-groonga/issues). Feel free to report your feedback.
```
Comment (cosmo0920) [inline en/_posts/2016-02-29-grafana-datasource-plugin-groonga.md:47]: 動作の対象主を省いてしまうと不完全な文になってしまいます。"perfect sentence"を目指したいですね。
あと、Feel free to ...で気軽に〜してください、という意味になります。

**Comment by cosmo0920** [comment]
I've commented where I feel hard to explain why this changes are needed in commit message.

**Comment by myokoym** [comment]
ありがとうございます。後で見ます。

**Comment by myokoym** [comment]
Thanks for your review. I've merged it.

**Comment by cosmo0920** [comment]
Thanks!

**Pull request merged**